### DISCUSSION TITLE
Windows build problem

### DISCUSSION THREAD

**Discussion by rpitre-wetdog** [maplibre/maplibre-native#982] [General]
I found a few problem in the build process. One I was able to fix (I think) since the build process did not finished so I am not able to test.:
1- In vcpkg_acquire_msys.cmake, line 346, msys2-runtime-3.2.0-8-x86_64.pkg.tar.zst no longer exist, you need to use amore recent package
2- This one I am not sure how to fix , find_package(CURL CONFIG REQUIRED) fails. I get this error message:
' Could not find a configuration file for package "CURL" that is compatible
  with requested version "".

  The following configuration files were considered but not accepted:

    C:/Users/rpitre/source/repos/maplibre/maplibre-gl-native/platform/windows/vendor/vcpkg/installed/x86-windows/share/curl/CURLConfig.cmake, version: 7.87.0-DEV (32bit)'

**Comment by etnav** [comment]
1. Seems to be fixed by: https://github.com/microsoft/vcpkg/pull/30546 


Error I am getting. haven't looked into it further thus far. But may just need vcpkg update as stated in linked PR.
```
error: Failed to download from mirror set
error: https://repo.msys2.org/msys/x86_64/msys2-runtime-3.2.0-8-x86_64.pkg.tar.zst: failed: status code 404
error: https://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/msys2-runtime-3.2.0-8-x86_64.pkg.tar.zst: failed: status code 404
error: https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/msys2-runtime-3.2.0-8-x86_64.pkg.tar.zst: failed: status code 404
error: https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/msys2-runtime-3.2.0-8-x86_64.pkg.tar.zst: failed: status code 404
error: https://mirrors.ustc.edu.cn/msys2/msys/x86_64/msys2-runtime-3.2.0-8-x86_64.pkg.tar.zst: failed: status code 404
error: https://mirror.bit.edu.cn/msys2/msys/x86_64/msys2-runtime-3.2.0-8-x86_64.pkg.tar.zst: failed: status code 502
error: https://mirror.selfnet.de/msys2/msys/x86_64/msys2-runtime-3.2.0-8-x86_64.pkg.tar.zst: failed: status code 404
error: https://mirrors.sjtug.sjtu.edu.cn/msys2/msys/x86_64/msys2-runtime-3.2.0-8-x86_64.pkg.tar.zst: failed: status code 404
```

**Reply by etnav** [reply]
@tdcosta100 Is this part cached or something and thus not encountered?

**Reply by tdcosta100** [reply]
@etnav, maybe vcpkg buildsystem has something hardcoded and needs update. I will do that ASAP.

**Reply by tdcosta100** [reply]
The PR #1073 is alive to fix this. If I understand correctly, this new version always points to the last MSYS2 files, so it won't be a problem anymore.
